netlink-packet-route/src/rtnl/link/nlas/link_infos.rs

```diff
@@ -714,6 +724,137 @@ impl<'a, T: AsRef<[u8]> + ?Sized> Parseable<NlaBuffer<&'a T>> for InfoVxlan {
     }
 }
 
+// https://elixir.bootlin.com/linux/v5.9.8/source/net/ipv4/ipip.c#L598
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub enum InfoIpTun {
+    Unspec(Vec<u8>),
+    Link(u32),
+    Local(Vec<u8>),
+    Remote(Vec<u8>),
+    Ttl(u8),
+    Tos(u8),
+    Proto(u8),
+    Pmtudisc(u8),
+    EncapType(u16),
+    EncapFlags(u16),
+    EncapSport(u16),
+    EncapDport(u16),
+    CollectMetadata(u8),
```

Comment:
The kernel has `if (nla_put_flag(skb, IFLA_IPTUN_COLLECT_METADATA))` which indicate this is a boolean with 0 size in data.